guibot/region.py

```diff
@@ -750,21 +714,18 @@ def click_expect(self, click_image_or_location, expect_target,
                 if i == retries - 1:
                     raise error
 
-    def click_vanish(self, click_image_or_location, expect_target,
-                     modifiers=None, timeout=60, retries=3):
+    def click_vanish(self, click_image_or_location: Image | Location,
+                     expect_target: str | Target, modifiers: list[str] = None,
+                     timeout: int = 60, retries: int = 3) -> "Match":
```

Comment:
This must be `Region`

Reply:
which one should be Region be here? the return?